universal-login-sdk/lib/integration/http/RelayerApi.ts

```diff
@@ -1,9 +1,9 @@
-import {http, HttpFunction, PublicRelayerConfig, RelayerRequest} from '@universal-login/commons';
+import {http, HttpFunction, PublicRelayerConfig, RelayerRequest, ApplicationInfo} from '@universal-login/commons';
 import {fetch} from './fetch';
 
 export class RelayerApi {
   private readonly http: HttpFunction;
-  constructor(relayerUrl: string, private applicationName: string) {
+  constructor(relayerUrl: string, private applicationInfo: ApplicationInfo) {
```

Comment:
This is terrible place to set `applicationInfo`.
It shoud be set up in sdk configuration. SDK class should pass it as an argument to connect frunction of RelayerAPi.

Reply:
We set up these informations in sdk config, and after it we create RelayerApi (sdk class) with applicationInfo, which use in connect and deploy api requests to relayer

Reply:
Still, it ApplicationInfo should not be passed to RelayerAPI.